R/system.R

```diff
@@ -368,6 +368,58 @@ getMongoCollectionNumberOfRows <- function(host = NULL, port = "", database = ""
   return(result)
 }
 
+#' Returns a AWS Athena connection.
+#' @export
+getAWSAthenaConnection <- function(driver = "", region = "", authenticationType = "IAM Credentials", s3OutputLocation = "", user = "", password = "", additionalParams = "", ...) {
+  loadNamespace("RODBC")
+  loadNamespace("DBI")
```

Comment:
Do we need DBI when using RODBC?

Reply:
DBI is actually not used so removed this loadNamespace for DBI.